src/libcore/iter/range.rs

```diff
@@ -420,5 +420,5 @@ impl<A: Step> DoubleEndedIterator for ops::RangeInclusive<A> {
     }
 }
 
-#[unstable(feature = "fused", issue = "35602")]
+#[stable(feature = "fused", since = "1.25.0")]
```

Comment:
`RangeInclusive` is still unstable so this should be `#[unstable(feature = "inclusive_range", reason = "recently added, follows RFC", issue = "28237")]`.